pvlib/test/test_solarposition.py

```diff
@@ -352,6 +352,18 @@ def test_get_solarposition_no_kwargs(expected_solpos):
     expected_solpos = np.round(expected_solpos, 2)
     ephem_data = np.round(ephem_data, 2)
     assert_frame_equal(expected_solpos, ephem_data[expected_solpos.columns])
+    
+
```

Comment:
needs the ``@requires_ephem`` decorator (see other tests for examples)

Reply:
done!